src/6_branch.js

```diff
@@ -508,7 +518,7 @@ Branch.prototype['init'] = wrap(
 			}
 		};
 
-		if (sessionData && sessionData['session_id'] && !link_identifier) {
+		if (sessionData && sessionData['session_id'] && !link_identifier && !utils.getParamValue('branchify_url')) {
```

Comment:
this is to trigger a v1/open everytime there's a `branchif_url`

Reply:
let's double check what happens when you have a _branch_match_id and a branchify_url as a query param, what would take precedence in this case? Would link data be returned from branch_match_id or from the branchify_url?

Reply:
confirmed `branch_match_id` takes precedence here. https://91szw-alopez.branchbeta.link/3vGThQ3XCS goes to http://192.168.192.251:8000/example.html?branchify_url=https%3A%2F%2F91szw-alopez.branchbeta.link%2Fnobranchlink&utm_medium=marketing&_branch_match_id=546068215899985938 but I have the `https://91szw-alopez.branchbeta.link/3vGThQ3XCS` only